bindings/py/tests/connections_test.py

```diff
@@ -1,498 +0,0 @@
-#!/usr/bin/env python
-# ----------------------------------------------------------------------
-# Numenta Platform for Intelligent Computing (NuPIC)
-# Copyright (C) 2017, Numenta, Inc.  Unless you have an agreement
-# with Numenta, Inc., for a separate license for this software code, the
-# following terms and conditions apply:
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero Public License version 3 as
-# published by the Free Software Foundation.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-# See the GNU Affero Public License for more details.
-#
-# You should have received a copy of the GNU Affero Public License
-# along with this program.  If not, see http://www.gnu.org/licenses.
-#
-# http://numenta.org/licenses/
-# ----------------------------------------------------------------------
-
-"""Unit tests for connections classes"""
-
-import numpy as np
-import unittest
-
-from nupic.bindings.math import Random, SparseMatrixConnections
-
-class ConnectionsTest(unittest.TestCase):
```

Comment:
Connections class is heavily used, this test should stay (if not just moved elsewhere)

Reply:
Sorry, my bad. Not an issue then